### DISCUSSION TITLE
MAA是怎么定位技能释放按钮的位置的呢？

### DISCUSSION THREAD

**Discussion by SherkeyXD** [MaaAssistantArknights/MaaAssistantArknights#1235] [General]
如题，我想制作一个能在wsa内使用快捷键的ahk脚本，但是在定位技能释放按钮的位置上遇到了问题。我只能截取某个特定干员的技能图标来进行定位，请问maa是用同样的方法进行定位的吗？
万分感谢

**Comment by MistEO** [comment]
你识别这个小闪电就好了

![BattleSkillReadyOnClick](https://user-images.githubusercontent.com/18511905/179790492-8ee2db95-3eef-4410-a2ef-0b6af9e6c2ab.png)

**Reply by SherkeyXD** [reply]
感谢，之前想过这个，没想到真的可以（
